Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -166,34 +166,51 @@ private func backingReward(fromProject project: Project) -> Reward? {
 }
 
 private func localizedDescription(project: Project, reward: Reward) -> String {
+  let goRewardlessEnabled = featureGoRewardlessIsEnabled()
```

Comment:
If we just did an early return at the top of this function we shouldn't have to make any other changes and would be easier to remove (hopefully) later on:

```swift
if featureGoRewardlessIsEnabled(), reward.isNoReward, !userIsBacking(reward: reward, inProject: project) {
  return Strings.This_holiday_season_support_a_project_for_no_reward()
}
```